themes/src/main/resources/theme/base/login/resources/js/authChecker.js

```diff
@@ -0,0 +1,76 @@
+
+function kcAuthChecker() {
+    var authChecker = this;
+
+    // How often to check if KEYCLOAK_SESSION cookie was added in the other browser tab?
+    var checkIntervalMillisecs = 2000;
```

Comment:
For constants we prefer to use `SCREAMING_SNAKE_CASE` (:snake:), also the [`const`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const) statement can be used, as the value will never change.
```suggestion
    const CHECK_INTERVAL_MILLISECS = 2000;
```